docs/src/increase-observability/metrics.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Monitor metrics
+weight: 5
+description: See all of the live infrastructure metrics available to give you an overview of resource usage.
+aliases:
+  - /dedicated/architecture/metrics.html
+---
+
+Platform.sh projects are accompanied by live infrastructure metrics that provide an overview of resource usage for environments.  
+
+Within the console, metrics can be found for an environment under **Metrics**:
+
+![Metrics dashboard](/images/metrics/all.png "0.65")
+
+The information under **Metrics** shows [usage metrics](#examples-of-how-to-read-metrics) for:
+
+* [Dedicated environments](../dedicated/overview/_index.md):
+  each of the three hosts in your [N+1 configuration](../dedicated/architecture/_index.md) and their average
+* Grid environments: for the average of your service and app containers
+
+## Default thresholds
+
+All of the graphs show labels for the following thresholds:
+
+* Usage that crosses _80%_ results in a **warning** label.
+* Usage that crosses _90%_ results in a **critical** label.
+* On Grid environments only, usage that crosses _100%_ results in a **burst** label.
```

Comment:
Change to " Usage that crosses _100%_ results in a **burst** label (concerning Grid environments only_)."
Note: The burst capability is available for containerized environments
  and allows a container to get more resources than it's allocated.
  Burst is considered useful for infrequent activities that cause usage spikes,
  but these additional resources aren't guaranteed.